<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>12</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 4px solid red;
	border-radius: 10px;
	background-color: qlineargradient(spread:pad, x1:0.995025, y1:1, x2:0, y2:0, stop:0 rgba(90, 198, 255, 255), stop:1 rgba(255, 255, 255, 255));
	font: 75 12pt &quot;MS Shell Dlg 2&quot;;
	color: red;
}

QLineEdit:focus {
	background:qlineargradient(spread:pad, x1:0.523, y1:0, x2:0.507, y2:1, stop:0 rgba(158, 99, 255, 255), stop:1 rgba(85, 255, 151, 255));
}</string>
   </property>
   <property name="placeholderText">
    <string>C:/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browse">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>360</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 12pt &quot;Palatino Linotype&quot;;
	color: red;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(190, 164, 224, 255), stop:1 rgba(255, 255, 255, 255));
	border: 4px solid red;
	border-radius: 10px;
}

QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0.995025, y1:1, x2:0, y2:0, stop:0 rgba(93, 255, 0, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
   </property>
   <property name="text">
    <string>BROWSE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setupbtn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>530</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 15pt &quot;Palatino Linotype&quot;;
	color: red;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(190, 164, 224, 255), stop:1 rgba(255, 255, 255, 255));
	border: 4px solid red;
	border-radius: 10px;
}

QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0.995025, y1:1, x2:0, y2:0, stop:0 rgba(93, 255, 0, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
   </property>
   <property name="text">
    <string>SETUP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="finish">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>530</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 15pt &quot;Palatino Linotype&quot;;
	color: red;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(190, 164, 224, 255), stop:1 rgba(255, 255, 255, 255));
	border: 4px solid red;
	border-radius: 10px;
}

QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0.995025, y1:1, x2:0, y2:0, stop:0 rgba(93, 255, 0, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
   </property>
   <property name="text">
    <string>FINISH</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>86</y>
     <width>761</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Palatino Linotype</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28pt &quot;Palatino Linotype&quot;;
color: red;
background-color: qlineargradient(spread:pad, x1:0.995025, y1:1, x2:0, y2:0, stop:0 rgba(90, 198, 255, 255), stop:1 rgba(255, 255, 255, 255));
border: 4px solid red;
border-radius: 10px;</string>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="tabChangesFocus">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>    SETUP HACKING PROGRAM
		 CREATED BY HDT</string>
   </property>
   <property name="backgroundVisible">
    <bool>false</bool>
   </property>
   <property name="centerOnScroll">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>741</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{
	border: 4px solid red;
	border-radius: 10px;
	background-color: qlineargradient(spread:pad, x1:0.995025, y1:1, x2:0, y2:0, stop:0 rgba(90, 198, 255, 255), stop:1 rgba(255, 255, 255, 255));
	color: red;
}
QProgressBar::chunk {
	background:qlineargradient(spread:pad, x1:0.523, y1:0, x2:0.507, y2:1, stop:0 rgba(158, 99, 255, 255), stop:1 rgba(85, 255, 151, 255));
	border-radius: 5px;
}
</string>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="format">
    <string>                                                   %p%</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(164, 224, 199, 255), stop:1 rgba(255, 255, 255, 255))</string>
   </property>
  </widget>
  <zorder>widget</zorder>
  <zorder>lineEdit</zorder>
  <zorder>browse</zorder>
  <zorder>setupbtn</zorder>
  <zorder>finish</zorder>
  <zorder>plainTextEdit</zorder>
  <zorder>progressBar</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
